src/test/groovy/graphql/nadel/engine/NadelExecutionStrategyTest.groovy

```diff
@@ -2424,6 +2512,9 @@ fragment F1 on TestingCharacter {
         def issue1Result = [id: "ISSUE-1", authorIds: ["USER-1", "USER-2"], __typename: "Issue"]
         def issue2Result = [id: "ISSUE-2", authorIds: ["USER-3"], __typename: "Issue"]
         resultData(response) == [issues: [issue1Result, issue2Result]]
+
+        resultComplexityAggregator.getTotalNodeCount() == 13
+        resultComplexityAggregator.getServiceNodeCountsMap().get("Issues") == 13
```

Comment:
Although you will probably on use it in testing, it might be a help to put a conviencence method on the class

`resultComplexityAggregator.getServiceNodeCounts("Issues") == 13`

which is a convenience for 

`resultComplexityAggregator.getServiceNodeCountsMap().get("Issues") == 13`

Reply:
ps.  Some people dont like that (eg there is no end on how many convenience methods you can have) - I do like it and it makes the tests easier to read